app/controller/sdl/RCModulesController.js

```diff
@@ -0,0 +1,946 @@
+     */
+    updateModuleSeatLocationContent: function() {
+      if (1 >= Object.keys(this.seatKeyLabelMapping).length) {
+        return;
+      }
+
+      var vehicle_representation =
+        SDL.SDLModelData.vehicleSeatRepresentation[FLAGS.VehicleEmulationType];
+      var driver_location = vehicle_representation[0];
+      var module_apps_mapping = {};
+
+      Object.keys(this.seatKeyLabelMapping).forEach(
+        key => {
+          module_apps_mapping[key] = 0;
+      });
+
+      SDL.SDLModel.data.registeredApps.forEach(app => {
+        var app_user_location = driver_location;
+        if (app.userLocation) {
+          app_user_location = app.userLocation;
+        }
+
+        var module_name =
+          SDL.VehicleModuleCoverageController.getModuleKeyName(app_user_location);
+        module_apps_mapping[module_name]++;
+      });
+
+      var new_content = []
+      Object.keys(this.seatKeyLabelMapping).forEach(
+        key => {
+          var new_name = this.getUserFriendlyModuleName(key);
+          if (module_apps_mapping[key] > 0) {
+            new_name = "[" + module_apps_mapping[key] + "] " + new_name;
+          }
+          this.seatKeyLabelMapping[key] = new_name;
+          new_content.push(new_name);
+      });
+
+      SDL.ControlButtons.RCInfo.RCModules.set('content', new_content);
+    },
+
+    /**
+     * @function fillSeatLocationCapabilities
+     * @description Function to generate seat locations capabilities
+     * @param {Array} representation
+     */
+    fillSeatLocationCapabilities: function(representation) {
+      var seat_capability = {};
+
+      var max_col_index =
+        SDL.VehicleModuleCoverageController.getVehicleMaxIndex(representation, 'col');
+      seat_capability['columns'] =
+        SDL.VehicleModuleCoverageController.getVehicleItemValue(
+          representation[max_col_index], 'col') + 1;
+
+      var max_row_index =
+        SDL.VehicleModuleCoverageController.getVehicleMaxIndex(representation, 'row');
+      seat_capability['rows'] =
+        SDL.VehicleModuleCoverageController.getVehicleItemValue(
+          representation[max_row_index], 'row') + 1;
+
+      var max_level_index =
+        SDL.VehicleModuleCoverageController.getVehicleMaxIndex(representation, 'level');
+      seat_capability['levels'] =
+        SDL.VehicleModuleCoverageController.getVehicleItemValue(
+          representation[max_level_index], 'level') + 1;
+
+      seat_capability['seats'] = [];
+      representation.forEach(seat => {
+        seat_capability.seats.push(
+          {
+            'grid': seat
+          }
+        );
+      });
+
+      SDL.remoteControlCapabilities.seatLocationCapability = seat_capability;
+    },
+
+    /**
+     * @function getModuleCurrentId
+     * @description getter for a readable module name to display on UI
+     * @param {String} module_id
+     */
+    getModuleCurrentId: function(module_type, module_id) {
```

Comment:
The second identifier used for the `moduleUUIDMapping` is a location grid coordinate as a string `L<level>R<row>C<col>` so `module_id` should be renamed to 
```suggestion
    getModuleCurrentId: function(module_type, location_name) {
```

Reply:
Also I tried to find where this function is being called but couldn't find it being used anywhere else in the code. If this  is an unused function it should probably be removed

Reply:
@ShobhitAd good catch. This function is not used in the latest implementation so it was removed in https://github.com/smartdevicelink/sdl_hmi/commit/3356c1d8aa1ec7b881f8a7e25bb4cde2a50033af